.gitignore

```diff
@@ -21,3 +21,5 @@ logs
 *.log
 
 gradle.properties
+LibrariesTest.kt
```

Comment:
Is this required?

Reply:
This is a little convention I use to test various external libraries